hasura/migrations/default/1663180519908_contributions_decouple_epochs/up.sql

```diff
@@ -0,0 +1,17 @@
+DROP TABLE IF EXISTS "public"."contributions";
+
+CREATE TABLE "public"."contributions"
+(
+  "id" bigserial NOT NULL,
+  "circle_id" bigint NOT NULL,
+  "user_id" bigint NOT NULL,
+  "description" text NOT NULL,
+  "datetime_created" timestamptz NOT NULL DEFAULT now(),
+  "created_at" timestamptz NOT NULL DEFAULT now(),
+  "updated_at" timestamptz NOT NULL DEFAULT now(),
+  "deleted_at" timestamptz,
+  PRIMARY KEY ("id"),
+  FOREIGN KEY ("circle_id") REFERENCES "public"."circles"("id") ON UPDATE restrict ON DELETE restrict,
+  FOREIGN KEY ("user_id") REFERENCES "public"."users"("id") ON UPDATE restrict ON DELETE restrict,
+  UNIQUE ("id")
+);
```

Comment:
We still need a trigger here to update the updated_at right? 

Like this: https://github.com/coordinape/coordinape/blob/main/hasura/migrations/default/1662391842396_interaction_events/up.sql#L15

Reply:
```suggestion
);

CREATE TRIGGER "set_public_contributions_updated_at" BEFORE
UPDATE
  ON "public"."contributions"
  FOR EACH ROW
   EXECUTE PROCEDURE "public"."set_current_timestamp_updated_at"
   ();

COMMENT ON TRIGGER "set_public_contributions_updated_at" ON "public"."contributions"
IS 'trigger to set value of column "updated_at" to current timestamp on row update';
```